Gruntfile.js

```diff
@@ -36,6 +36,16 @@ module.exports = function(grunt) {
 		dateString: now.substr(0, now.indexOf("T")),
 
 		pkg: grunt.file.readJSON("package.json"),
+		commonDtsFileCreation: grunt.file.write(require("path").join("lib", "common", ".d.ts"), ""),
```

Comment:
I think you can just do `"lib/common/.d.ts"` - it should work on Windows as well.